tracy-client/src/state.rs

```diff
@@ -1,4 +1,5 @@
 use crate::Client;
+#[cfg(all(feature = "enable", feature = "manual-lifetime"))]
```

Comment:
Might be cleaner to use the module delegate pattern for all these CFGs. So create two `mod imp`s and have `struct Client(imp::Client)` so that you can have a single cfg on the mod declaration.

Reply:
I doubt the amount of complexity here right now is sufficiently high for a strategy proposed to pay off.